PREVIEW.md

```diff
@@ -1342,9 +1428,9 @@ public class Variables {
         int castVariable = (int) (4 / 2);
     }
 
-    public void localVariables(
-
-    ) {final List<Filter> filterList = new ArrayList<>();
+    public void localVariables() {final List<
```

Comment:
the body should start in a new line

Reply:
see #38, I already reported the problem :)

Reply:
ok im gonna merge, I can live with this then, and we can get it into master. It's not worth then before so we should merge it :)